### ISSUE TITLE
Arc browser tabs remain in list of open apps in AltTab menu

### ISSUE THREAD

**Issue by ChrisPetalotis** [lwouis/alt-tab-macos#3641]
**Describe the bug**

It doesn't always happen, but sometimes when using the Arc browser, old tabs, that have been previously closed, appear in the list of open applications in the AltTab menu. When selecting one of those tabs, the Arc browser opens at the last state I left it, regardless of whether the selected tab is still open or not (meaning I don't necessarily end up on the selected webpage from the AltTab menu).

I am currently on MacOS Sequoia, where this behaviour is more prominent when it happens. I had the same issue on MacOS Sonoma too, but fewer tabs remained in the list of recent applications when that happened.

The way to "fix" this is to completely quit the Arc app and re-open it.

**Screenshot**
![image](https://github.com/user-attachments/assets/dfa4b162-0ce7-4212-b44e-d3b2e3ffc148)

**Steps to reproduce the bug**
Not sure what triggers it, so I don't have a clear way to reproduce this behaviour.

**Your environment**

* AltTab version: 6.72.0
* macOS version: MacOS Sequoia

**Comment by lwouis** [comment]
Hi,
Symptoms are similar to https://github.com/lwouis/alt-tab-macos/issues/3589
Could you please have a look?
Thank you 🙇

**Comment by ChrisPetalotis** [comment]
Yes, I had the issue reported in #3589 too. That was addressed after quitting the Amazon Q app and was hoping the issue reported here would go as well. However, this one persists even when the Amazon Q app is closed.
